packages/network/test/network-misc.test.js

```diff
@@ -32,7 +32,7 @@ const provideDurableZone = key => {
   return zone.subZone(key);
 };
 
-test('handled protocol', async t => {
+test.failing('handled protocol', async t => {
```

Comment:
what is the plan for these failing tests?

Reply:
these changes are no longer part of this PR, so marking this resolved